universal-application-tool-0.0.1/app/views/BaseHtmlView.java

```diff
@@ -110,18 +110,27 @@ protected ContainerTag renderPaginationDiv(
 
   protected ContainerTag renderSearchForm(
       Http.Request request, Optional<String> search, Call searchCall) {
+    return renderSearchForm(request, search, searchCall, Optional.empty(), Optional.empty());
+  }
+
+  protected ContainerTag renderSearchForm(
+      Http.Request request,
+      Optional<String> search,
+      Call searchCall,
+      Optional<String> htmlClasses,
```

Comment:
Are these called anywhere with non empty values? I'm not seeing it at least.

Reply:
aaah maybe not, is it alright to leave them as is?